app/bundles/CoreBundle/Views/Default/pageheader.html.php

```diff
@@ -19,20 +19,23 @@
         </div>
         <div class="col-xs-7 col-sm-6 col-md-7 va-m">
             <div class="toolbar text-right" id="toolbar">
-                <?php $view['slots']->output('actions'); ?>
 
-                <div class="toolbar-bundle-buttons pull-left"><?php $view['slots']->output('toolbar'); ?></div>
-                <div class="toolbar-form-buttons hide pull-right">
-                    <div class="btn-group toolbar-standard hidden-xs hidden-sm "></div>
-                    <div class="btn-group toolbar-dropdown hidden-md hidden-lg">
-                        <button type="button" class="btn btn-default btn-main"></button>
-                        <button type="button" class="btn btn-default btn-nospin  dropdown-toggle" data-toggle="dropdown"
-                                aria-expanded="false"><i class="fa fa-caret-down"></i></button>
-                        <ul class="dropdown-menu dropdown-menu-right" role="menu"></ul>
+                    <div class="pull-left"><?php $view['slots']->output('toolbar'); ?></div>
```

Comment:
This change seemed to have messed up the themes toolbar. 
![2016-11-21_03-09-33](https://cloud.githubusercontent.com/assets/63312/20480628/325f9580-af98-11e6-9a98-df49b77ff4e7.png)